content/en/user-guide/aws/redshift/index.md

```diff
@@ -1,51 +1,128 @@
+
+{{< command >}}
+$ REDSHIFT_URL=$(awslocal redshift describe-clusters \
+      --cluster-identifier $REDSHIFT_CLUSTER_IDENTIFIER | jq -r '(.Clusters[0].Endpoint.Address) + ":" + (.Clusters[0].Endpoint.Port|tostring)')
+{{< / command >}}
+
+### Create a Glue database, connection, and crawler
+
+You can create a Glue database using the [`CreateDatabase`](https://docs.aws.amazon.com/glue/latest/webapi/API_CreateDatabase.html) API. The following command will create a Glue database:
+
+{{< command >}}
+$ awslocal glue create-database \
+      --database-input "{\"Name\": \"$GLUE_DATABASE_NAME\"}"
+{{< / command >}}
+
+You can create a connection to the RedShift cluster using the [`CreateConnection`](https://docs.aws.amazon.com/glue/latest/webapi/API_CreateConnection.html) API. The following command will create a Glue connection with the RedShift cluster:
+
+{{< command >}}
+$ awslocal glue create-connection \
+      --connection-input "{\"Name\":\"$GLUE_CONNECTION_NAME\", \"ConnectionType\": \"JDBC\", \"ConnectionProperties\": {\"USERNAME\": \"$REDSHIFT_USERNAME\", \"PASSWORD\": \"$REDSHIFT_PASSWORD\", \"JDBC_CONNECTION_URL\": \"jdbc:redshift://$REDSHIFT_URL/$REDSHIFT_DATABASE_NAME\"}}"
+{{< / command >}}
+
+Finally, you can create a Glue crawler using the [`CreateCrawler`](https://docs.aws.amazon.com/glue/latest/webapi/API_CreateCrawler.html) API. The following command will create a Glue crawler:
+
+{{< command >}}
+$ awslocal glue create-crawler \
+      --name $GLUE_CRAWLER_NAME \
+      --database-name $GLUE_DATABASE_NAME \
+      --targets "{\"JdbcTargets\": [{\"ConnectionName\": \"$GLUE_CONNECTION_NAME\", \"Path\": \"$REDSHIFT_DATABASE_NAME/%/$REDSHIFT_TABLE_NAME\"}]}" \
+      --role r1
+{{< / command >}}
+
+### Create table in RedShift
+
+You can create a table in RedShift using the [`CreateTable`](https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_TABLE_NEW.html) API. The following command will create a table in RedShift:
+
+{{< command >}}
+$ REDSHIFT_STATEMENT_ID=$(awslocal redshift-data execute-statement \
+      --cluster-identifier $REDSHIFT_CLUSTER_IDENTIFIER \
+      --database $REDSHIFT_DATABASE_NAME \
+      --sql \
+  "create table $REDSHIFT_TABLE_NAME(salesid integer not null, listid integer not null, sellerid integer not null, buyerid integer not null, eventid integer not null, dateid smallint not null, qtysold smallint not null, pricepaid decimal(8,2), commission decimal(8,2), saletime timestamp)" | jq -r .Id)
+{{< / command >}}
+
+You can check the status of the statement using the [`DescribeStatement`](https://docs.aws.amazon.com/redshift-data/latest/APIReference/API_DescribeStatement.html) API. The following command will check the status of the statement:
+
+{{< command >}}
+$ wait "awslocal redshift-data describe-statement \
```

Comment:
@HarshCasper: is this `wait` a built-in cli tool in Linux? It doesn't work on mac (syntax is wrong, the `wait` is expecting a job).

Reply:
Yeah — https://linux.die.net/man/3/wait